examples/07_advanced/plot_surface_bids_analysis.py

```diff
@@ -136,9 +157,35 @@
         surf_mesh=fsaverage5["inflated"],
         stat_map=stat_map,
         hemi=hemi,
+        title=f"(language-string), {hemi} hemisphere ; scipy",
+        colorbar=True,
+        cmap="bwr",
+        threshold=1.96,
+        bg_map=fsaverage_data,
+    )
+
+
+# %%
+# Use SecondLevel
```

Comment:
@bthirion  
Any idea why using scipy and Second level model would give such different results?
Am I missing something obvvious?

## OLD: pure scipy

![Figure_1_scipy](https://github.com/user-attachments/assets/dd32c832-28d4-4118-bdbd-5cc5b1c80813)

## NEW: Nilearn Second level model

![Figure_3_nl](https://github.com/user-attachments/assets/f775d5da-6b49-4fbc-a1b0-e2d16461623b)

Once this discrepancy I would prefer to only keep the nilearn second level model appraoch in this example.